config/samples/network_v1alpha1_bgpconf.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: network.kubesphere.io/v1alpha1
+kind: BgpConf
+metadata:
+  name: bgpconf-sample
+spec:
+  # Add fields here
+  as : 65000
+  router-id : 192.168.0.2
```

Comment:
`router-id` should be named as camelCase. So do `peer-as` `neighbor-address` `add-paths` `send-max`

Reply:
ok